rag/flow/tests/test_token_chunker.py

```
@@ -277,6 +290,73 @@ def test_json_delimiter_mode_pdf_positions_retained():
+        param = token_chunker_module.TokenChunkerParam()
+        param.delimiter_mode = "token_size"
+        param.check()
+        assert param.delimiter_mode == "delimiter"
+
+        bad = token_chunker_module.TokenChunkerParam()
+        bad.delimiter_mode = "nope"
+        try:
+            bad.check()
+            raise AssertionError("expected check() to reject unknown delimiter_mode")
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)